src/main/java/uk/gov/companieshouse/search/api/service/search/impl/alphabetical/AlphabeticalSearchRequestService.java

```diff
@@ -22,365 +30,105 @@
-            "COMPANY)LIMITED",
-            "LTD.CO.",
-            "& COMPANY.LIMITED",
-            "THE)LIMITED",
-            "&& CO.LIMITED",
-            "COMPANY UN-LIMITED",
-            "CO.PLC",
-            "COMPANY",
-            "AND COMPANY.LIMITED",
-            "& CO;LIMITED",
-            "CO",
-            "& CO..LIMITED",
-            "COMPANY",
-            "AND COMPANYLIMITED",
-            "CO",
-            "COMPANY",
-            "LTD",
-            "PLC.",
-            "& CO LTD.",
-            "& COMPANY LTD.",
-            "SE",
-            "CCG CYF",
-            "CCG CYFYNGEDIG",
-            "CWMNI CELL GWARCHODEDIG",
-            "AR CWMNI PAC",
-            "AR CWMNI CYF",
-            "AR CWMNI CCC",
-            "CWMNI CYFYNGEDIG",
-            "CYFYNGEDIG CWMNI",
-            "& CWMNI CYFYNGEDIG",
-            "+CWMNI CYFYNGEDIG",
-            "AR CWMNI CYFYNGEDIG",
-            "PARTNERIAETH CYFYNGEDIG",
-            "CWMNI BUDDIANT CYMUNEDOL CCC",
-            "CWMNI CYFYNGEDIG CYHOEDDUS",
-            "CWMNI BUDDIANT CYMUNEDOL C.C.C",
-            "AR CWMNI CYFYNGEDIG CYHOEDDUS",
-            "CWMNI BUDDIANT CYMUNEDOL",
-            "PARTNERIAETH ATEBOLRWYDD CYFYNGEDIG",
-            "CWMNIBUDDSODDIANTPENAGORED",
-            "CWMNI CWMNI CYFYNGEDIG CYHOEDDUS",
-            "&CWMNI CWMNI CYFYNGEDIG CYHOEDDUS",
-            "+CWMNI CWMNI CYFYNGEDIG CYHOEDDUS",
-            "CNI PARTNERIAETH ATEBOLRWYDD CYFYNGEDIG",
-            "& CNI PARTNERIAETH ATEBOLRWYDD CYFYNGEDIG",
-            "+ CNI PARTNERIAETH ATEBOLRWYDD CYFYNGEDIG",
-            "AR CWMNI CWMNI CYFYNGEDIG CYHOEDDUS",
-            "AR CNI PARTNERIAETH ATEBOLRWYDD CYFYNGEDIG",
-            "CWMNI PARTNERIAETH ATEBOLRWYDD CYFYNGEDIG",
-            "& CWMNI PARTNERIAETH ATEBOLRWYDD CYFYNGEDIG",
-            "CBC",
-            "PAC",
-            "CYF",
-            "CCC",
-            "CBCN",
-            "CYF.",
-            "NULL",
-            "C.B.C",
-            "C.C.C",
-            "CNI PAC",
-            "C.C.C.",
-            "CWMNI",
-            "CYFYNGEDIG",
-            "& CWMNI",
-            "+ CWMNI",
-            "ANGHYFYNGEDIG",
-            "AR CNI PAC",
-            "AR CWMNI",
-            "CWMNI PAC",
-            "CWMNI CYF",
-            "CWMNI CCC",
-            "& AR CNI PAC",
-            "& CWMNI PAC",
-            "& CWMNI CCC",
-            "+ CWMNI PAC",
-            "+ CWMNI CYF",
-            "+ CWMNI PARTNERIAETH ATEBOLRWYDD CYFYNGEDIG",
-            "CWMNIBUDDSODDIACHYFALAFNEWIDIOL",
-            "AR CWMNI PARTNERIAETH ATEBOLRWYDD CYFYNGEDIG",
-            "CWMNI BUDDIANT CYMUNEDOL CYHOEDDUS CYFYNGEDIG",
-            "+CWMNI CCC"
-    };
 
     /**
      * {@inheritDoc}
      */
     @Override
-    public SearchRequest createSearchRequest(String corporateName, String requestId) {
+    public SearchResults createSearchRequest(String corporateName, String requestId) throws SearchException {
 
```

Comment:
I think the function of this method may have changed, but its name not been changed to reflect that? If so, log message may need updating too.

Reply:
Good spot!